.rspec

```diff
@@ -1,3 +1,4 @@
 --format documentation
 --color
 --require spec_helper
+--backtrace
```

Comment:
I'm not sure that making "print full backtraces" the default is a great idea.  Typically in my projects I make it an env-var-settable option, with something like this in `spec_helper.rb`:

```ruby
RSpec.configure { |c| c.full_backtrace = ENV.key?("RSPEC_FULL_BACKTRACE") }
```